csharp-selenium-webdriver-sample/SampleTest.cs

```diff
@@ -6,57 +6,72 @@
 using System;
 using System.IO;
 using Selenium.Axe;
-using OpenQA.Selenium.Chrome;
 
 namespace CSharpSeleniumWebdriverSample.Test
 {
     [TestClass]
-    [DeploymentItem("SamplePage.html")]
+    //  Install chromedriver to enable Chrome broswer and copy it to the same directory as the deployed test assemblies.
     [DeploymentItem("chromedriver.exe")]
+    //  Install geckodriver to enable Firefox brower and copy it to the same directory as the deployed test assemblies.
     [DeploymentItem("geckodriver.exe")]
     public class SampleTest
     {
+        //  Intance to control the browser.
         private IWebDriver _webDriver;
 
         [TestCleanup]
         public virtual void TearDown()
         {
+            //  You should quit the driver on TearDown which close every associated window.
             _webDriver?.Quit();
         }
 
 
         [TestMethod]
+        // Test on Chrome
         [DataRow(BrowserType.Chrome)]
+        //  Test on Firefox
         [DataRow(BrowserType.Firefox)]
         public void RunScanOnGivenElement(BrowserType browser)
         {
+            //  Intilaize the driver based on the browser type
             this.InitDriver(browser);
+            //  Load the test page
             LoadTestPage();
 
+            //  Analyze the unorder list element in the page, and get the results
             AxeResult results = RunScanOnGivenElementBySelector("ul");
+            //  Assert that the number of vilaztions that were found is that what you expect
             Assert.AreEqual(3, results.Violations.Length);
         }
 
         private AxeResult RunScanOnGivenElementBySelector(string elementSelector)
         {
+            //  Find the first element matching the element selector parameter
             var selectedElement = _webDriver.FindElement(By.TagName(elementSelector));
 
+            //  Instruct the driver to analyze the current element, and return the result as AxeResult object
             return _webDriver.Analyze(selectedElement);
         }
 
 
         private void LoadTestPage()
         {
+            //  For simplicity, we're pointing our test browser directly to a static html file on disk.
             string integrationTestTargetFile = Path.GetFullPath(@"SamplePage.html");
             string integrationTestTargetUrl = new Uri(integrationTestTargetFile).AbsoluteUri;
 
+            //  Instructs the driver to navigate the browser to the target URL, this should start the brower and open the page in a new window tab
             _webDriver.Navigate().GoToUrl(integrationTestTargetUrl);
         }
 
         private void InitDriver(BrowserType browser)
         {
+            //  Create a web driver based on the bowser type which is ehither Chrome of Firefox
```

Comment:
>   [](start = 14, length = 2)

It looks like the "2 spaces after the slashes" is an intentional pattern. I'm not sure if that's part of the style in the sample-pipelines repo, but it's not a pattern that we follow in AIWin...